ui/webui/src/components/storage/MountPointMapping.jsx

```diff
@@ -73,18 +73,20 @@ const getInitialRequests = (usablePartitioningRequests, requiredMountPoints) =>
     const rootOriginalRequest = usablePartitioningRequests.find(r => r["mount-point"] === "/");
     const bootEfiOriginalRequest = usablePartitioningRequests.find(r => r["mount-point"] === "/boot/efi");
 
+    console.log(requiredMountPoints);
```

Comment:
I think you want to log also some string to explain what this is about. Otherwise it's useless clutter in the log.

Reply:
Oops, that wasn't supposed to stay there. I guess I need some script to check for forgotten debug prints :-)